src/test/java/io/confluent/connect/elasticsearch/ElasticsearchClientTest.java

```diff
@@ -106,7 +106,7 @@ public void setup() {
   @After
```

Comment:
lets add a test for creating a datastream and for creating a datastream that already exists